examples/bernoulli/bernoulli.stan

```diff
@@ -6,6 +6,10 @@ parameters {
   real<lower=0,upper=1> theta;
 }
 model {
-  theta ~ beta(1,1);  // uniform prior on interval 0,1
-  y ~ bernoulli(theta);
+  profile("gahh") {
```

Comment:
clean me?!

Reply:
Yeah sorry so fyi I just have these in here for easy testing locally but once the Stan PR is merged and we are about to merge this I'm going to delete all these examples